### ISSUE TITLE
Delay for a second pinging a new member with DM off

### ISSUE THREAD

**Issue by JaJe92** [plexidev/issue-tracker#22]
Hello!
I've setup my community channel to have everyone acces to a channel named "verify" where a new user can type !verify anytime.
I've setup that channel to NOT read the message history to always be clean for these new users.

However, I've noticed an issue for new members which have DM OFF, the bot automatically pings them to that channel but they're unable to see the message because they cannot read the message history. 

A fix suggestion: Can you delay the ping the new member with a second later after they joined so they can see the message?
Basically the bot is now pinging new new member too fast than they can join the server and are unable to see the history message of the bot.

**Comment by Zorotic** [comment]
This is something of where sometimes it works and other times it doesn't, I'll look into fixing this within the next update.
